server/app/forms/QuestionForm.java

```diff
@@ -192,4 +198,46 @@ public final boolean isUniversal() {
   public final void setIsUniversal(boolean universal) {
     this.isUniversal = universal;
   }
+
+  public final ImmutableSet<PrimaryApplicantInfoTag> primaryApplicantInfoTags() {
+    return this.primaryApplicantInfoTags;
+  }
+
+  // The names of these functions must match up to the fieldName value in the
+  // PrimaryApplicantInfoTag
```

Comment:
(blocking) Can you give more info on why the names must match?

Also make it a javadoc comment and add it to each method. That way down the road the ones below the comment don't get lost + easier to see in a PR.